php-practice/section5.php

```diff
@@ -320,3 +338,25 @@ public function send() {
  *   'cost' => 3.99
  * ];
  */
+
+class ShoppingCart {
+  private $item;
+  private $shoppingCart;
+
+  $item = [
+    'name' => 'Pringles',
+    'cost' => 3.99
+  ];
```

Comment:
That does make sense. But for the purposes of our class, I'm not going to do that correct? Because I want to be able to add multiple items to the shopping cart using the addItem function. So that is something I would code outside of the class, correct?